doc/user/zebra.rst

```diff
@@ -680,6 +680,68 @@ FRR's cli or frr.conf or zebra.conf. This section shows how
 to configure SRv6 on FRR. Of course SRv6 can be used as standalone,
 and this section also helps that case.
 
+.. clicmd:: show segment-routing srv6 [json]
+
+   This command dumps the SRv6 information configured on zebra. The SRv6
+   information includes the encapsulation parameters (e.g., the IPv6 source
+   address used for the encapsulated packets) and the SRv6 locators configured
+   on zebra.
+
+   The following example shows a router with two SRv6 locators ``loc1`` and
+   ``loc2``::
+
+      router# sh segment-routing srv6
+      Encapsulation:
+      Source-Address: fc00:0:1::1
+      Locator:
+      Name                 ID      Prefix                   Status
+      -------------------- ------- ------------------------ -------
+      loc1                       1 fc00:0:1::/48            Up
```

Comment:
just a nit: why is this aligned to the right, while others - left?

Reply:
@ton31337 Because I missed a "-" in the `vty_out()` call :)

My intention is to align everything to the left, I think it looks nicer.
I updated both the VTY output and the documentation. Thanks.